lib/stronger_parameters/constraints/boolean_constraint.rb

```diff
@@ -2,7 +2,7 @@
 
 module StrongerParameters
   class BooleanConstraint < Constraint
-    TRUE_VALUES  = [true, 'true', '1', 1].freeze
+    TRUE_VALUES  = [true, 'true', '1', 1, 'on'].freeze
     FALSE_VALUES = [false, 'false', '0', 0].freeze
```

Comment:
unchecked checkboxes don't send anything when the form is submitted, it's up to the dev to create a hidden input above the checkbox and explicitly give it a value afaik